zebra-chain/src/sprout/tree.rs

```diff
@@ -84,7 +96,7 @@ pub struct Position(pub(crate) u64);
 /// each treestate.
 #[derive(Clone, Copy, Default, Eq, PartialEq, Serialize, Deserialize, Hash)]
 #[cfg_attr(any(test, feature = "proptest-impl"), derive(Arbitrary))]
-pub struct Root([u8; 32]);
+pub struct Root(pub [u8; 32]);
```

Comment:
I made this public again because the wrapped field is accessed here https://github.com/ZcashFoundation/zebra/blob/4f050ac5ffd3be6c0431fecd5936fc0a840a6673/zebra-chain/src/sprout/tests/tree.rs#L134

Reply:
This is also public in the existing Sapling implementation for the same reason.

Reply:
Since they both have `From` implementations you could replace those with e.g. 

```rust
tree::Root::from(**sprout_roots.get(&height).expect("test vector exists"));
```

or

```rust
**sprout_roots.get(&height).expect("test vector exists").into();
```

(though `into()` that seems a bit less readable in that particular context.)

That seems out of scope for this ticket though, so feel free to create a ticket to do it later if you prefer.

Reply:
It was a quick change for the new Sprout code. Thank you for the suggestion.

I've tried doing the same for Sapling and Orchard. I realized I'd have to change it on a lot of places, so I'm going to open a new issue.